src/wallet/rpc/spend.cpp

```diff
@@ -1729,8 +1729,12 @@ RPCHelpMan walletcreatefundedpsbt()
                     }
                                 },
                                 RPCExamples{
-                            "\nCreate a transaction with no inputs\n"
+                            "\nCreate a PSBT with a specified input and an OP_RETURN output:\n"
```

Comment:
We shouldn't be encouraging this in the first place. Suggest replacing the example with yours.

Reply:
I tend to agree, updated.